build.gradle

```diff
@@ -772,6 +772,12 @@ List<String> jacocoExclusions = [
         'org.opensearch.ad.AnomalyDetectorExtension.1.1',
         'org.opensearch.ad.EntityProfileRunner',
         'org.opensearch.ad.caching.CacheProvider',
+        'org.opensearch.ad.transport.ADResultBulkTransportAction',
+        'org.opensearch.ad.transport.ADResultBulkRequest',
+        'org.opensearch.ad.transport.ADResultBulkAction',
+        'org.opensearch.ad.ratelimit.ResultWriteRequest',
+        'org.opensearch.ad.AnomalyDetectorJobRunner.1',
+        'org.opensearch.ad.AnomalyDetectorJobRunner.2',
```

Comment:
Curious why these classes are not having test coverage?

Reply:
The classes regarding ADResultBulkTransportAction have been commented out for this PR temporarily, I will handle them in [this PR](https://github.com/opensearch-project/anomaly-detection/pull/876) to enable HCAD real time analysis, as the ADResultBulkTransportAction is used to index multi entity Anomaly Results. This current PR includes some preparational work for the HCAD workflow